add v0.8 release notes (#50)
opea-project/docs: release_notes/v0.8.md

# What’s New in OPEA v0.8

- Broaden functionality

>     - Supported frequently asked questions (FAQs) generation GenAI example

Author: It's title of PR, should we change it?

>     - Expanded the support of LLMs such as Llama3.1 and Qwen2 and supported LVMs such as llava

Author: It's title of PR, should we change it?

>     - Enabled end-to-end peformance and accuracy benchmarking

Author: It's title of PR, should we change it?

>     - Enabled end-to-end peformance and accuracy benchmarking

Reviewer: Looks good to me.

>     - Supported the experimental Agent microservice and GenAI example

Reviewer 2: ```suggestion
    - Supports the experimental Agent microservice and GenAI example
```

Author: It's title of PR, should we change it?

>     - Supported LLM serving on Ray

Author: It's title of PR, should we change it?

- Multi-platform support

>     - Released the docker images of GenAI components under OPEA dockerhub and supported the deployment with docker

Reviewer 2: Also, can you add a link to the OPEA dockerhub here:
```suggestion
    - Releases the Docker images of GenAI components under OPEA dockerhub and supports the deployment with Docker
```

Author: It's title of PR, should we change it?

>     - Supported the cloud native deployment through Kubernetes manifests and GenAI Microservices Connector (GMC)

Reviewer 2: ```suggestion
    - Supports cloud-native deployment through Kubernetes manifests and GenAI Microservices Connector (GMC)
```

Author: It's title of PR, should we change it?

>     - Enabled the experimental authentication and authorization support using JWT tokens

Reviewer 2: ```suggestion
    - Enables the experimental authentication and authorization support using JWT tokens
```

Author: It's title of PR, should we change it?

>     - Validated ChatQnA on multiple platforms such as Xeon, Gaudi, AIPC and Nvidia, and AWS

Reviewer 2: ```suggestion
    - Validates ChatQnA on multiple platforms such as Xeon, Gaudi, AIPC, Nvidia, and AWS
```

Author: It's title of PR, should we change it?

# OPEA Highlights

> - In this release, OPEA supports large vision models and provides an initial Agent microservice. It also integrates more components, such as vLLM on Ray, giving users more choices for different AI functionalities. We have expanded the application range, enabling ChatQnA on AIPC and validating it on AWS. Additionally, more examples, such as FAQGen, are coming. Deployment is important, Kubernetes manifests for ChatQnA, DocSum, CodeGen and CodeTrans are created.

Reviewer 3: Can we add some highlights for GMC as well?  Like 
GenAI Microservices Connector (GMC) supports the launching, monitoring, and updating of GenAI microservice Applications on Kubernetes. It essentially supports a Kubernetes Custom Resource Definition for GenAI chains/pipelines that may be comprised of sequential, conditional, and parallel steps.

Reviewer 3: @mkbhanda @yipan ^^

Reviewer 4: Please add, based on @irisdingbj input 
"GenAI Microservices Connector (GMC) supports the launching, monitoring, and updating of GenAI microservice Applications on Kubernetes. It essentially supports a Kubernetes Custom Resource Definition for GenAI chains/pipelines that may be comprised of sequential, conditional, and parallel steps."

> - In this release, OPEA supports large vision models and provides an initial Agent microservice. It also integrates more components, such as vLLM on Ray, giving users more choices for different AI functionalities. We have expanded the application range, enabling ChatQnA on AIPC and validating it on AWS. Additionally, more examples, such as FAQGen, are coming. Deployment is important, Kubernetes manifests for ChatQnA, DocSum, CodeGen and CodeTrans are created.

Reviewer 4: Should we add that we tested OPEA pipeline works on Nvidia?

Author: good idea

> - In this release, OPEA supports large vision models and provides an initial Agent microservice. It also integrates more components, such as vLLM on Ray, giving users more choices for different AI functionalities. We have expanded the application range, enabling ChatQnA on AIPC and validating it on AWS. Additionally, more examples, such as FAQGen, are coming. Deployment is important, Kubernetes manifests for ChatQnA, DocSum, CodeGen and CodeTrans are created.

Author: remove coming

# GenAIExamples 
- ChatQnA
    - add ChatQnA instructions for AIPC([26d4ff](https://github.com/opea-project/GenAIExamples/commit/26d4ff11ffd323091d80efdd3f65e4c330b68840))

>     - Adapt Vllm response format ([034541](https://github.com/opea-project/GenAIExamples/commit/034541404e23ce3927c170237817e98f9323af26))

Author: remove all commits

    - Change tgi tag because gaudi driver is upgraded to 1.16.1 ([6796ef](https://github.com/opea-project/GenAIInfra/commit/6796ef2560645c59cdf7b09af9a2d8aa0cb0d5a5))
    - Update README for new manifests([ec32bf](https://github.com/opea-project/GenAIInfra/commit/ec32bf04459fdbb4c8f99ebd1bac3216ad2e5283))
    - Support multiple router service in one namespace([0ac732](https://github.com/opea-project/GenAIInfra/commit/0ac73213b501fb5949a5ac8bf7f52d5a4acef580)) 
    - Improve workflow trigger conditions to be more precise([ab5c8d](https://github.com/opea-project/GenAIInfra/commit/ab5c8d8c07d8f8353315b7ebaf1eb745bf7b28e5))
    - Remove unnecessary component DocSumGaudi which would cause error([9b973a](https://github.com/opea-project/GenAIInfra/commit/9b973aceb25c307f2c7692c9364ebac9040b9a5d))
    - Remove chart_test scripts and add script to dump pod status([88caf0](https://github.com/opea-project/GenAIInfra/commit/88caf0df947866ffe609cf60805282970f887429))
</details>

## Thanks to these contributors
This release includes valuable contributions from the Intel core team.. We would also like to thank everyone who reported issues or provided feedback. 

> Contributors

Reviewer 2: Is this a list of contributors across all the repos? I'm not sure we need this full list of 70 contributors, maybe just the top 10 (if any)?

Reviewer: I think it's ok to put the contributors here with the small icons.

# What’s New in OPEA v0.8
- Broaden functionality coverage
    - OPEA v0.8 Support LVM(Large Vision Models), it provides a microservice for llava on Intel Gaudi2. It also create an initial Agent microservice, OPEA now has the capability to process information, evaluate situations, and take actionable steps. Meanwhile more components are coming, such as vLLM on Ray, giving users more choices for different AI functionalities. FAQGen (Frequently Asked Questions Generation) is a new example of LLM application.
- Deploy across more platforms

>     - We validate ChatQnA on AWS, including GenAI Microservices Connector (GMC), Kubernetes manifests and docker. Meanwhile it can work on AIPC. Kubernetes manifests for ChatQnA, DocSum, CodeGen and CodeTrans are created. The document for deploying OPEA with Docker Hub is also included in this release. GMC supports the launching, monitoring, and updating of GenAI microservice Applications on Kubernetes.

Reviewer 4: s/validate/validated
s/AWS/AWS and Nvidia H100

Add one more sentence ..
This release also supports basic Authentication and Authorization using JWT tokens.

Author: not sure how to add AWS and Nvidia H100, current sentence is "OPEA pipeline can also operate on AIPC and Nvidia H100"
one more sentence has been added
